microservices/object_detection/detector_base.py

```diff
@@ -21,60 +21,66 @@
 from PIL import ImageDraw
 
 import detect
+from config import log
```

Comment:
Are you sure that by loading this variable like this, then the rest of the code in `config.py` after `log = logging.getLogger()` is actually executed? I admit I have limited knowledge on how these things work with python imports, but doesn't seem obvious to me, I'd doublecheck or just organise things differently.

Reply:
it works perfectly, but I am going to create a module logger with a class on it, so that we can reuse it\

doing so we can have multiple loggers with differents porposes  ...

Reply:
Ok, in that case my concerns were nonsense. I need to read a bit more how the loading is done, given it turns out I had a wrong idea. However, as you say, it will be much better to have a reusable class!